js/interface.js

```diff
@@ -653,7 +890,7 @@ $('.file-manager-wrapper')
     if (!folderName) {
       return;
     }
-
+    
```

Comment:
Empty lines should be completely empty, without spaces

Reply:
fixed